manifests/todo-app.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: todo-app
+  name: todo-app
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: todo-app
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: todo-app
+    spec:
+      containers:
+      - image: eu.gcr.io/reload-oliver-onboarding/todo-app:${RELEASE_TAG}
+        name: todo-app
+        env:
+        - name: MYSQL_HOST
+          value: todo-db-svc
+        - name: MYSQL_USER
+          value: root
+        - name: MYSQL_PASSWORD
+          value: secret
```

Comment:
Obviously, this would not fly in real life :wink: One (very simple) option is to use [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/), but the main thing gained is not having to commit passwords to `git`. It does not by default encrypt anything at rest, although [GKE does have additional features](https://cloud.google.com/kubernetes-engine/docs/how-to/encrypting-secrets) in this space that vanilla Kube does not.

Nothing to change here for now tho 🔑